TheInternet/HoversPage.cs

```diff
@@ -0,0 +1,51 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+
+namespace TheInternet;
+
+public class HoversPage : BasePage
+{
+    IList<IWebElement> userImagesList => driver.FindElements(By.CssSelector("div [class = 'figure']"));
+   
+    public HoversPage(IWebDriver driver) : base(driver)
+    {
+        this.driver = driver;
+    }
+
+    public UserCaption hoverOveruserImage(int userNumber)
+    {
+        userNumber = userNumber - 1;
```

Comment:
Language usage opportunity:

This is such a common operation that there's a shortcut for it:
```
  userNumber--;
```

Increment by one is done with `++`